db/migrate/20190923162900_create_edu_programs.rb

```diff
@@ -0,0 +1,22 @@
+class CreateEduPrograms < ActiveRecord::Migration
+  def change
+    create_table :edu_programs do |t|
+      t.string :facility_code, null: false
+      t.string :institution_name, null: false
+      t.string :school_locale, null: false
+      t.string :provider_website, null: false
+      t.string :provider_email_address, null: false
+      t.string :phone_area_code, null: false
+      t.string :phone_number, null: false
+      t.string :student_vet_group
+      t.string :student_vet_group_website, null: false
+      t.string :vet_success_name, null: false
+      t.string :vet_success_email, null: false
+      t.string :vet_tec_program
+      t.integer :tuition_amount, null: false
+      t.integer :length_in_weeks, null: false
+    end
+
+    add_index :edu_programs, [:facility_code, :vet_tec_program] , using: :btree
```

Comment:
will you generally be querying on `LOWER(vet_tec_program)` like here? if so, you may want to make  alter the index. https://github.com/department-of-veterans-affairs/gibct-data-service/pull/445/files#diff-79373e1323911953ef2804c7b52cebb8R554